scripts/config

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+source _common.sh
+source /usr/share/yunohost/helpers
+
+#=================================================
+
+ynh_app_config_apply() {
+    _ynh_app_config_apply
+
```

Comment:
You will need to reapply the new config as same as we do here: https://github.com/YunoHost-Apps/forgejo_ynh/blob/testing/scripts/config#L8